packages/jsii-rosetta/lib/util.ts

```diff
@@ -243,3 +243,15 @@ export async function pathExists(path: string): Promise<boolean> {
     throw err;
   }
 }
+
+const ANSI_PATTERN = new RegExp(
+  [
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]+)*|[a-zA-Z\\d]+(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-nq-uy=><~]))',
+  ].join('|'),
+  'g',
+);
+
+function stripColorCodes(x: string) {
+  return x.replace(ANSI_PATTERN, '');
+}
```

Comment:
> (Copied the regex from an MIT-licensed package to avoid adding extra dependencies)

Still, it would not hurt to attribute to the original package/authors?